php.ini

```diff
@@ -23,4 +23,9 @@ realpath_cache_ttl = 7200
 ;
 ; Multi store support
 ;
-auto_prepend_file = /app/public/magento-vars.php
\ No newline at end of file
+auto_prepend_file = /app/public/magento-vars.php
+
+:
+; This feature was DEPRECATED in PHP 5.6.0, and REMOVED as of PHP 7.0.0.
+:
+always_populate_raw_post_data = -1
```

Comment:
@pjcdawkins per Magento 2 docs it should be set:

http://devdocs.magento.com/guides/v2.0/install-gde/prereq/php-ubuntu.html

In other case we are getting errors in `var/.setup_cronjob_status`:

    "php_settings_verified": {
        "responseType": "error",
            "data": {
                "always_populate_raw_post_data": {
                "message": "Your PHP Version is 5.6.17-0+deb8u1, but always_populate_raw_post_data = 0.\n \t        $HTTP_RAW_POST_DATA is deprecated from PHP 5.6 onwards and will be removed in PHP 7.0.\n \t        This will stop the installer from running.\n\t        Please open your php.ini file and set always_populate_raw_post_data to -1.\n \t        If you need more help please call your hosting provider.",
                "helpUrl": "http://php.net/manual/en/ini.core.php#ini.always-populate-settings-data",
                "error": true
            }
        }
    }